### DISCUSSION TITLE
About Suspending Account；Ticket : #2970719

### DISCUSSION THREAD

**Discussion by wellingfun** [community/community#137918] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I submitted a support ticket through Github's support page but it's been more than 7 days and I still haven't received any response or support.
 
The following is the content of my appeal at that time.
-------------------------------------------------------------------------------
Dear Administrator

Greetings,
I am the owner of the GitHub account with the ID qiliq (1025913408@qq.com).
On August 28, 2024, between 15:39 and 15:48 UTC+8, I did not log in to GitHub. However, my account was compromised as hackers posted numerous replies containing virus-infected download links.I must clarify that none of the actions taken during this period were mine.After receiving numerous reply notification emails, I promptly logged into my GitHub account but was unable to determine how my account was compromised. Consequently, I had to revoke most of my permissions and encryption keys.Despite this, my actions were deemed in violation, leading to the suspension of my account. I hereby declare that none of the actions taken during the aforementioned period were mine. I appeal to the administrator to reinstate my account, pledging to use it more vigilantly to prevent any further breaches.
I deeply regret the unwanted repercussions stemming from the theft of my account and offer my sincerest apologies once again.I trust that the matter will be objectively investigated and my account reinstated.

Thank you.
-------------------------------------------------------------------------------
I don't know what caused my account to be hacked and an unknown website address was sent, all of which happened without my knowledge.
<img width="1128" alt="1725620958971" src="https://github.com/user-attachments/assets/db48385c-e14d-4258-8553-b184f68e7bf8">
<img width="1136" alt="1725620972976" src="https://github.com/user-attachments/assets/e0eb80e2-1b7b-447c-9448-677ec7945a72">
<img width="1137" alt="1725621136053" src="https://github.com/user-attachments/assets/d442ff66-667d-47ec-90c0-5b381740cb50">
<img width="1121" alt="1725621636299" src="https://github.com/user-attachments/assets/18f3a2e3-ab30-47e5-9cb8-d2e2de03c556">

My entire work has been suspended and my repositories are no longer accessible. There is no way to restore or at least what caused this terrible situation from github, which makes me worried about my future creations if this situation continues.

**Comment by wellingfun** [comment]
有人吗

**Comment by devKanix** [comment]
My github account was also flagged how can i fix that?

**Reply by raushankumar1511** [reply]
you can use github discussion

**Comment by wellingfun** [comment]
I have been making a request for a month now, but I still haven't received a response

**Comment by FranPratama** [comment]
Take your time bud, I have submitted an appeal as well in 26th July 2024 but still no updates yet to this day 🙃

**Comment by wellingfun** [comment]
Can someone help me? What should I do

**Comment by shinybrightstar** [comment]
Hi @wellingfun 

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
